### ISSUE TITLE
[APP SUBMITTED]: database disk image is malformed

### ISSUE THREAD

**Issue by copekyle** [SiCKRAGETV/sickrage-issues#1314]
### INFO
Python Version: **2.7.9 (default, Dec 10 2014, 12:28:03) [MSC v.1500 64 bit (AMD64)]**
Operating System: **Windows-7-6.1.7601-SP1**
Branch: **master**
Commit: SiCKRAGETV/SickRage@281409d366b228a455879e02b2d55d3967db29e4
Link to Log: https://gist.github.com/c035f6b5e6181728977a
### ERROR
```
Thread-19 :: Fatal error executing query: database disk image is malformed
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by copekyle** [comment]
I was having issues with my SR not snatching anything.  It seems the logs have been changed and so I can't see specifics about the search/snatch process when viewing the db log.  I thought it would be a good time to back up my copy of sickrage and start fresh.  SO I used the built in backup/restore, downloaded a fresh version of the master branch and used the restore.  All my shows appear but when ever I try a manual search or try to click on a show, I get the "database malformed" error.
